examples/cfrl_mnist.ipynb

```diff
@@ -0,0 +1,732 @@
+{
```

Comment:
<p>Is this intended to be a markdown cell and not an executable one?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/457/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>